build

```diff
@@ -109,3 +126,14 @@ for project in filter(
     module = module.lstrip("lib") if module.startswith("lib") else module
     dst_file = os.path.join(QUERY_MODULES_DIRECTORY, module)
     shutil.copy2(src_file, dst_file)
+
+#################################################
+#            Optional copy dist folder          #
+#################################################
+
+arguments = get_arguments()
+
+if arguments.path is not None:
+    print('Copying files from folder {src} to {dst} folder.'.format(src=QUERY_MODULES_DIRECTORY, dst=arguments.path))
+    copy_dist(arguments.path)
+    print('Copying done!')
```

Comment:
This structure didn't make much sense to me, but I continued with what was already there. I would suggest that we add 
 main() function and that this part getArguments() is first part and then copy at the end.

Reply:
Reason why it was written as a script was mimicking the bash script. If more functionalities are to be adapted here, we might benefit from  reconstruction

Reply:
Makes sense